cli/config.go

```diff
@@ -15,14 +15,18 @@ import (
 	"cogentcore.org/core/base/logx"
 )
 
+// TheMetaConfig holds the current [MetaConfig] data for access
+// of verbose and quiet state from client apps.
+var TheMetaConfig *MetaConfig
```

Comment:
As I previously told you, you can obtain the log level through `logx.UserLevel`. Unexport everything and use that in Cogent Author where you need it.